GSA-TTS/FAC
# 3708 address historical federal awards with missing additional award identification

## Description
Some historical reports failed the first round of migration due to missing additional award identification. Upon investigation, it was discovered that the failure was due to invalid characters associated with the additional identification (value contains =""516084" instead of 516084) in the audit.AWARDIDENTIFICATION field in the database. This prevented the code from properly generating the awards workbook and hence from properly validating the data. The solution applied here was to add logic that sanitizes the audit.AWARDIDENTIFICATION values by removing leading `=""` and trailing `" ` for Excel.

## How to test
1. Launch the stack from the main branch.
2. Identify a valid report with non empty `audit.AWARDIDENTIFICATION`  values.
3. Change any `audit.AWARDIDENTIFICATION`  field value by adding `=""` and trailing `"` (for instance, change `123` to `=""123"``.
4. Attempt to migrate this report and notice how it fails due to "missing additional award identification"
5. Switch to this PR branch and re-run the migration for the same report and observe that it processes properly.

For extra points, you may try to create from scratch an audit report and submit it from this PR branch to confirm that process works as expected.

## PR checklist: submitters


## PR checklist: reviewers


The larger the PR, the stricter we should be about these points.
